src/components/Book.js

```diff
@@ -1,25 +1,51 @@
 import './Book.css';
-import { useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import BookInfo from './bookInfo/BookInfo';
 import SingleBook from './SingleBook/SingleBook';
+import { getBooks } from '../redux/books/booksSlice';
 
 const Book = () => {
-  const books = useSelector((state) => state.books.books);
+  const {
+    books, isLoading, error, done,
+  } = useSelector((state) => state.books);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getBooks());
```

Comment:
- [ ] Good job adding loading the books from the API using the lines below, however when you add a book it should be rendered directly on the screen , It requires the user to reload the page, so he may see the added (or deleted ) book. The dependency you use does not allow you to get the real-time changes 👍   Kindly change the dependency from `[dispatch, done]` into `[books]`

```bash
  useEffect(() => {
    dispatch(getBooks());
  }, [books]);
```

Check [this link](https://blog.bitsrc.io/understanding-dependencies-in-useeffect-7afd4df37c96) for more about dependency 😉

Reply:
Thank you for the review. I have implemented the recommended change